bx_py_utils/stacktrace.py

```diff
@@ -0,0 +1,135 @@
+"""
+Code in this module is largely from https://github.com/jazzband/django-debug-toolbar,
```

Comment:
The license of [django-debug-toolbar](https://github.com/jazzband/django-debug-toolbar/blob/master/LICENSE) is incompatible with this project. Can we either depend on them as a library or rewrite the code ourselves?

Reply:
Initially djdt was a dependency, but @jedie brough up the idea of decoupling it. Which makes sense if you want to use this in a production environment, where djdt most likely will not be installed.

Reply:
Pushed a new version that does not relate to djdt in any way.